docs/_guide/07-tools.md

```diff
@@ -0,0 +1,194 @@
+---
+layout: post
+title: Tools
+slug: tools
+---
+
+{::options toc_levels="1..3" /}
+* ToC
+{:toc}
+
+
+lit-html is available from the npm registry. If you're already using npm to manage dependencies, you can use lit-html much like any other JavaScript library you install from npm. This section describes some additional tools or plugins you might want to add to your workflow to make it easier to work with lit-html.
+
+lit-html is delivered as a set of JavaScript modules. If you aren't already using JavaScript modules in your project, you may need to add a couple of steps to your development and build workflow.
+
+
+## Setup
+
+The simplest way to add lit-html to a project is to install it from the npm registry. 
+
+1. If you're starting a brand-new project, run the following command in your project area to set up npm:
+
+    ```bash
+    npm init
+    ```
+
+    Respond to the prompts to set up your project. You can hit return to accept the default values.
+
+2. Install lit-html.
+
+    ```bash
+    npm i lit-html
+    ```
+
+
+3. If you're working on a project with many front-end dependencies, you may want to use the npm  `dedupe` command to try and reduce duplicated modules:
+
+    `npm dedupe` 
+
+## Development
+
+During the development phase, you might want the following tools:
+
+* IDE plugins, for linting and code highlighting.
+* Linter plugins, for checking lit-html templates.
+* A dev server, for previewing code without a build step.
+
+
+### IDE plugins
+
+There are a number of IDE plugins that may be useful when developing with lit-html. In particular, we recommend using a code highlighter that works with lit-html style templates. In addition, we recommend using a linter like ESLint that supports modern JavaScript.
+
+VSCode plugin
+
+* https://marketplace.visualstudio.com/items?itemName=bierner.lit-html
+
+TypeScript plugin (works with Sublime and Atom)
+
+* https://github.com/Microsoft/typescript-lit-html-plugin
+
+More plugins
+
+The [awesome-lit-html](https://github.com/web-padawan/awesome-lit-html#ide-plugins) repo lists other IDE plugins.
+
+
+### Linter plugins
+
+ESLint is recommended for linting lit-html code. The following ESLint plugin can be added to check for some common issues in lit-html templates:
+
+* [https://github.com/43081j/eslint-plugin-lit](https://github.com/43081j/eslint-plugin-lit)
+
+### Dev server
+
+
+lit-html is packaged as JavaScript modules. Many developers prefer to import modules using node-style module identifiers, which aren't supported yet by browsers. To run in the browser, these module identifiers need to be transformed to browser-ready module identifiers. The Polymer dev server, which is part of the Polymer CLI, performs this transformation on the fly, so you can preview your project during development without a build step. Another alternative is the Open Web Components Dev Server (`owc-dev-server`).
+
+If you're using webpack for your build process, you can also use the webpack dev server.
```

Comment:
Since Rollup can output modules, I greatly prefer to reference it as the example bundler.